### PR TITLE
Simplify `F.swish` test

### PR DESCRIPTION
**Pull request by ishanrai05**
Simplifies `test_swish` by using `testing.FunctionTestCase`

Fixes part of issue #6071

### CODE REVIEW

**Comment by ishanrai05** [comment]
@toslunar  The `double backward tests` seem to fail for some reason.  Can you please review the code?  Thanks already :)

**Comment by toslunar** [comment]
@ishanrai05 Your code looks as good as `F.swish` should pass the new test.  In `chainer/functions/activation/swish.py`, `SwishGrad.backward` (i.e. the code for double backprop) needs to be fixed to treat the cases only one of two `grad_outputs` (`ggx`, `ggb`) is given.

Would you like to bugfix `F.swish`? / Shall I take over the PR?

**Comment by ishanrai05** [comment]
@toslunar  I could try if you don't mind. Will help me to understand the code in a better way. I'll definitely ask for your help whenever required. :)

**Comment by toslunar** [comment]
`DivGrad.backward` is an example of handling `None` in `grad_outputs`.  Good luck!
https://github.com/chainer/chainer/blob/4a2361a3258d55ca7bbc0d72c0e4a22efcc2cf85/chainer/functions/math/basic_math.py#L445-L479

**Comment by ishanrai05** [comment]
Hi @toslunar . I found the error and fixed it.  Could you please review the code again?

File: tests/chainer_tests/functions_tests/activation_tests/test_swish.py [hunk 1]
```diff
@@ -22,73 +28,49 @@
         {'dtype': numpy.float64},
     ]
 ))
-@testing.fix_random()
-class TestSwish(unittest.TestCase):
+# @testing.fix_random()
```
Comment (toslunar) [inline tests/chainer_tests/functions_tests/activation_tests/test_swish.py]: Could you remove the line?

Comment (ishanrai05) [reply tests/chainer_tests/functions_tests/activation_tests/test_swish.py]: @toslunar  Done

**Comment by toslunar** [comment]
Jenkins, test this please.

**Comment by toslunar** [comment]
The Jenkins failures
```
tests.chainer_tests.functions_tests.activation_tests.test_elu.TestELU_use_chainerx_false__chainerx_device_None__use_cuda_true__cuda_device_1__use_cudnn_never__cudnn_deterministic_false__autotune_false__cudnn_fast_batch_normalization_false__use_ideep_never_param_0_{alpha_range=(-2.0,  0.0), dtype=<class 'numpy.float16'>, shape=(3,  2)}.test_double_backward
tests.chainer_tests.functions_tests.activation_tests.test_elu.TestELU_use_chainerx_false__chainerx_device_None__use_cuda_true__cuda_device_1__use_cudnn_never__cudnn_deterministic_false__autotune_false__cudnn_fast_batch_normalization_false__use_ideep_never_param_0_{alpha_range=(-2.0,  0.0), shape=(3, 2), dtype=<type  'numpy.float16'>}.test_double_backward
tests.chainer_tests.functions_tests.array_tests.test_spatial_transformer_sampler.TestSpatialTransformerSamplerConsistencyWithCuDNN_param_0_{dtype=<class  'numpy.float16'>}.test_consistency_with_cudnn_cpu
```
are unrelated to the PR.

@ishanrai05 While the PR is ready to be merged as it is, the commit date (Thu Mar 14 15:04:08 2019 +0000) looks too new.  Just FYI, check the clock or the timezone of your environment.

**Comment by ishanrai05** [comment]
@toslunar I checked. Time and dates are correct in my environment although the time zone is incorrec 0530 hrs behind. 
The new time maybe there because I did `git commit --amend` to make the changes. I use it because the commit history looks much cleaner that way. :)

**Pull request merged**